src/many-server/src/server.rs

```diff
@@ -326,11 +329,28 @@ impl LowLevelManyRequestHandler for Arc<Mutex<ManyServer>> {
                     LowLevelManyRequestHandler::execute(fb.as_ref(), envelope).await
                 }
                 (None, None) => {
-                    let response = ResponseMessage::error(
-                        &cose_id.identity,
-                        id,
-                        ManyError::could_not_route_message(),
-                    );
+                    let this = self.lock().unwrap();
```

Comment:
Let's meet about this. I don't think this is the right way of implementing this. There shouldn't be any changes to `server.rs` IMO.